plans/install_published.pp

```diff
@@ -27,17 +27,13 @@
 #   Installs the application from an airgap bundle.
 # @param wait_for_app
 #   Whether or not to wait for app deployment to complete before returning.
-# @param system_ready_wait_minutes
-#   Number of minutes to wait for all k8s system pods to be ready after
-#   node restart.
 plan pam_tools::install_published(
   TargetSpec $targets,
   String $license_file,
   Optional[String[6]] $password = undef,
   Optional[String] $config_file = undef,
   Optional[String] $airgap_bundle = undef,
   Boolean $wait_for_app = true,
-  Integer $system_ready_wait_minutes = 5,
```

Comment:
Should this have been passed to `pam_tools::wait_for_app`?

Reply:
The wait_for_app task has three separate timeouts. We haven't exposed those to date. The system_ready_wait_minutes was only used for restart_k8s so far (which is over in the kurl_test::install_replicated_app).